File: MPI/mpicheck/tests/functions/2.0/MPI_Alloc_mem.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Aint var_0;
    MPI_Info var_1;
    void *var_2;
    int ret;
    /* calls */
    ret = MPI_Alloc_mem(var_0, var_1, var_2);
    ret = PMPI_Alloc_mem(var_0, var_1, var_2);
    return 0;
}

File: MPI/mpicheck/tests/functions/2.0/MPI_Alloc_mem.f

        
        program main
        include 'mpif.h'
        
        INTEGER(KIND=MPI_ADDRESS_KIND) var_0
       INTEGER var_1
       INTEGER(KIND=MPI_ADDRESS_KIND) var_2
       INTEGER var_3
        call mpi_alloc_mem(var_0, var_1, var_2, var_3)
       call pmpi_alloc_mem(var_0, var_1, var_2, var_3)
        end program main
    